### PR TITLE
Update errormessage for unsupported device identifieres

### LINKED ISSUE #260: Small typo in `GENERIC_422_UNSUPPORTED_DEVICE_IDENTIFIERS` - example
**Problem description**
Incorrect reading in `"Supported device supported are: ..."`

```yaml
            GENERIC_422_UNSUPPORTED_DEVICE_IDENTIFIERS:
              summary: None of the provided device identifiers is supported by the implementation
              description: Message may list the supported device identifiers
              value:
                status: 422
                code: UNSUPPORTED_DEVICE_IDENTIFIERS
                message: "Supported device supported are: ..."
```

**Expected behavior**
```yaml
message: "Supported device identifiers are: ..."
```

**Alternative solution**

Without listing the supported device identifiers.
```
            GENERIC_422_UNSUPPORTED_DEVICE_IDENTIFIERS:
              summary: None of the provided device identifiers is supported by the implementation
              description: Message identifies that none of the provided device identifiers are supported
              value:
                status: 422
                code: UNSUPPORTED_DEVICE_IDENTIFIERS
                message: None of the provided device identifiers is supported by the implementation
```

### PR DESCRIPTION
**Pull request by maxl2287**
#### What type of PR is this?

Add one of the following kinds:

* correction

#### What this PR does / why we need it:
Fixing a typo / mistake in the error for `GENERIC_422_UNSUPPORTED_DEVICE_IDENTIFIERS`

#### Which issue(s) this PR fixes:

Fixes #260

### CODE REVIEW

**Review by bigludo7 — DISMISSED** [review]
LGTM
From a process perspective does it means that we have to do a new release @jlurien ?

**Review by jlurien — COMMENTED** [review]
For any change in the specs in the main branch we have to set versions to wip again

**Comment by jlurien** [comment]
> LGTM From a process perspective does it means that we have to do a new release @jlurien ?

We have to decide if changes are worthy for a new release. I would wait until we have something really relevant, and in that case update all small things as well

**Comment by maxl2287** [comment]
@jlurien & @bigludo7 I updated the version to "wip".
I guess we can then integrate it into the main, right? (if you are okay with it)

**Review by jlurien — APPROVED** [review]

**Comment by jlurien** [comment]
> @jlurien & @bigludo7 I updated the version to "wip". I guess we can then integrate it into the main, right? (if you are okay with it)

I noticed that you have to update as well the servers url to /vwip

File: code/API_definitions/location-retrieval.yaml [hunk 1]
```diff
@@ -87,7 +87,7 @@ info:
     # Further info and support
 
     (FAQs will be added in a later version of the documentation)
-  version: 0.3.0
+  version: wip
```
Comment (akoshunyadi) [inline code/API_definitions/location-retrieval.yaml:90]: also the server url should be set to wip: 
- url: '{apiRoot}/location-retrieval/vwip'
the same for location-verification

**Review by jlurien — APPROVED** [review]
LGTM

**Pull request merged**